### PR TITLE
PLP another set of validation methods

### PR DESCRIPTION
**Pull request by abeeralia2017**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/ibatta/SelFW/pull/13%23discussion_r153200944%22%2C%20%22https%3A//github.com/ibatta/SelFW/pull/13%23discussion_r153205846%22%2C%20%22https%3A//github.com/ibatta/SelFW/pull/13%23discussion_r153207136%22%5D%2C%20%22comments%22%3A%20%7B%22Pull%2018b3f9ce779e997403fd1e512ad97db4b70232ff%20src/com/generic/util/SelectorUtil.java%2023%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/ibatta/SelFW/pull/13%23discussion_r153200944%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22please%20explain%20%22%2C%20%22created_at%22%3A%20%222017-11-27T13%3A51%3A35Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/18301319%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ibatta%22%7D%7D%2C%20%7B%22body%22%3A%20%22I%20assigned%20the%20%5C%22value%5C%22%20to%20the%20tempVal%20then%20made%20a%20check%20on%20the%20original%20value%20if%20it%20contains%20%5C%22pressEnter%5C%22%20then%20take%20the%20first%20split%20of%20it%20and%20assign%20it%20to%20the%20tempVal%20and%20then%20write%20the%20value%20needed%20and%20press%20the%20Enter%20key.%5Cr%5CnPlease%20note%20this%20change%20is%20needed%20for%20search%20inputs%20in%20which%20we%20enter%20a%20value%20then%20press%20the%20Enter%20key.%22%2C%20%22created_at%22%3A%20%222017-11-27T14%3A11%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/33716221%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/abeeralia2017%22%7D%7D%2C%20%7B%22body%22%3A%20%22ok%20%5Cr%5Cn%22%2C%20%22created_at%22%3A%20%222017-11-27T14%3A17%3A02Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars2.githubusercontent.com/u/18301319%3Fv%3D4%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/ibatta%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20src/com/generic/util/SelectorUtil.java%3AL256-270%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-Pull 18b3f9ce779e997403fd1e512ad97db4b70232ff src/com/generic/util/SelectorUtil.java 23'></a> &nbsp;<b><a href='https://github.com/ibatta/SelFW/pull/13#discussion_r153200944'>File: src/com/generic/util/SelectorUtil.java:L256-270</a></b>
- <a href='https://github.com/ibatta'></a> please explain
- <a href='https://github.com/abeeralia2017'></a> I assigned the "value" to the tempVal then made a check on the original value if it contains "pressEnter" then take the first split of it and assign it to the tempVal and then write the value needed and press the Enter key.
Please note this change is needed for search inputs in which we enter a value then press the Enter key.
- <a href='https://github.com/ibatta'></a> ok

<a href='https://www.codereviewhub.com/ibatta/SelFW/pull/13?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/ibatta/SelFW/pull/13?approve=1'></a>&nbsp;<a href='https://github.com/ibatta/SelFW/pull/13'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by ibatta** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/ibatta/SelFW/pull/13'>Refresh</a>.<a href='#crh-update-none'></a>

File: src/com/generic/util/SelectorUtil.java [hunk 1]
```diff
@@ -254,7 +256,14 @@ public static String doAppropriateAction(Map <String, Object> webElementInfo ) t
 						  logs.debug(MessageFormat.format(LoggingMsg.WRITING_TO_SEL, "", value, byAction.toString()));
 						  WebElement field = getDriver().findElement(byAction);
 						  field.clear();
-						  field.sendKeys(value);
+						  String tempVal = value;
```
Comment (ibatta) [inline src/com/generic/util/SelectorUtil.java]: please explain

Comment (abeeralia2017) [reply src/com/generic/util/SelectorUtil.java]: I assigned the "value" to the tempVal then made a check on the original value if it contains "pressEnter" then take the first split of it and assign it to the tempVal and then write the value needed and press the Enter key.
Please note this change is needed for search inputs in which we enter a value then press the Enter key.

Comment (ibatta) [reply src/com/generic/util/SelectorUtil.java]: ok

Comment (ibatta) [inline src/com/generic/util/SelectorUtil.java]: :+1:

**Comment by ibatta** [comment]
:shipit:

**Pull request merged**